model/workflow.go

```diff
@@ -130,6 +184,34 @@ func (w *Workflow) UnmarshalJSON(data []byte) error {
 	if err := json.Unmarshal(data, &workflowMap); err != nil {
 		return err
 	}
+
+	// if len(workflowMap["auth"]) > 0 {
```

Comment:
should we just delete this？

Reply:
oops, we should, I've forgot to delete :)